Azure/sonic-buildimage
# Add pcied to PMON docker to monitor the PCIe device status

- Why I did it
Sonic needs to monitor the pcie device status and alert any missing pcie device issue.
- How I did it
Add pcied to PMON
- How to verify it
Check pmon/syslog message to see if the status reports are correct.
- Description for the changelog
